pkg/apis/servicecatalog/types.go

```diff
@@ -70,6 +70,12 @@ type ClusterServiceBrokerSpec struct {
 
 	// RelistDuration is the frequency by which a controller will relist the
 	// broker when the RelistBehavior is set to ServiceBrokerRelistBehaviorDuration.
+	// Users are cautioned against configuring low values for the RelistDuration,
+	// as this can easily overload the controller manager in an environment with
+	// many brokers. The actual interval is intrinsically governed by the
+	// configured resync interval of the controller, which acts as a minimum bound.
+	// For example, with a resync interval of 5m and a RelistDuration of 2m, relists
```

Comment:
Why the mention of "resync interval"? Isn't that for resyncing with the informers not brokers?

Reply:
It is, and that governs the minimum period that we will evaluate the relist interval on.

Reply:
I'm fine with alternate wording if you can make a suggestion.

Reply:
ah, so no "RelistDuration" value means we default to the resync interval value?

Reply:
But when a Relist is specified then resync has no impact, right?

Reply:
where in the code is the resync the minimum? I see where RelistDuration is used to determine it, but not resync. But perhaps I'm looking at the wrong spot.

Reply:
There's a behavior field that governs the mode.  If you use the interval mode, you have to specify a duration.  The other choice is manual.

The controller resync interval is a property of the controller and is set by a CLI flag.

Reply:
yea, I got that. I'm just looking for the code where it does the equivalent of `max(relist,resync)` - I'm only seeing the resync value taken into account.

Reply:
Sorry, I meant I'm only seeing where Relist is taken into account.

Reply:
The reason why the resync interval is a lower bound is because we won't even know the relist duration has passed until something kicks the controller to try to process the broker resource again. In the case of no spec changes, that would be a resync.

Reply:
@duglin So there's no actual code that does a check; it's an emergent behavior of the system.

Reply:
ah, thanks for the clarification

Reply:
Given that, I think it's a good thing to add this comment clarifying that relationship, and I think the comment does a good job clarifying this relationship to users without diving too deep into implementation details